src/window.jl

```diff
@@ -18,9 +18,19 @@ const window_defaults = @d(:url => "about:blank",
                            "use-content-size" => true,
                            :icon => Pkg.dir("Blink", "deps", "julia.png"))
 
+const windowhooks = Function[]
+
+windowinit(f) =
+    push!(windowhooks, f)
+
 function Window(a::AtomShell, opts::Associative = Dict())
   id = @js a createWindow($(merge(window_defaults, opts)))
-  return Window(id, a)
+  w = Window(id, a)
+  println("Exec hooks", windowhooks)
+  for f in windowhooks
+      f(w)
+  end
```

Comment:
Perhaps window hooks should be part of BlinkDisplay only, and called just after `loadhtml`?

For one thing `loadhtml` as currently defined will undo the display hooks. But more importantly, windows not booted by BlinkDisplay probably shouldn't be forced to load a bunch of display css/js.

(This is another good reason for specifying deps declaratively; each display can take only what it wants, when it wants it, rather than injecting everything everywhere)